pylint.conf

```diff
@@ -0,0 +1,665 @@
+        too-many-statements,
+        too-many-boolean-expressions,
+        consider-merging-isinstance,
+        too-many-nested-blocks,
+        simplifiable-if-statement,
+        redefined-argument-from-local,
+        no-else-return,
+        consider-using-ternary,
+        trailing-comma-tuple,
+        stop-iteration-return,
+        simplify-boolean-expression,
+        inconsistent-return-statements,
+        useless-return,
+        consider-swap-variables,
+        consider-using-join,
+        consider-using-in,
+        consider-using-get,
+        chained-comparison,
+        consider-using-dict-comprehension,
+        consider-using-set-comprehension,
+        simplifiable-if-expression,
+        no-else-raise,
+        unnecessary-comprehension,
+        consider-using-sys-exit,
+        no-else-break,
+        no-else-continue,
+        super-with-arguments,
+        pointless-statement,
+        expression-not-assigned,
+        attribute-defined-outside-init,
+        protected-access,
+        arguments-differ,
+        signature-differs,
+        fixme,
+        unused-argument,
+        redefined-outer-name,
+        cell-var-from-loop,
+        self-cls-assignment,
+        broad-except,
+        raise-missing-from,
+        apply-builtin,
+        basestring-builtin,
+        buffer-builtin,
+        cmp-builtin,
+        coerce-builtin,
+        execfile-builtin,
+        file-builtin,
+        long-builtin,
+        raw_input-builtin,
+        reduce-builtin,
+        standarderror-builtin,
+        unicode-builtin,
+        xrange-builtin,
+        coerce-method,
+        delslice-method,
+        getslice-method,
+        setslice-method,
+        no-absolute-import,
+        old-division,
+        dict-iter-method,
+        dict-view-method,
+        next-method-called,
+        metaclass-assignment,
+        indexing-exception,
+        raising-string,
+        reload-builtin,
+        oct-method,
+        hex-method,
+        nonzero-method,
+        cmp-method,
+        input-builtin,
+        round-builtin,
+        intern-builtin,
+        unichr-builtin,
+        map-builtin-not-iterating,
+        zip-builtin-not-iterating,
+        range-builtin-not-iterating,
+        filter-builtin-not-iterating,
+        using-cmp-argument,
+        eq-without-hash,
+        div-method,
+        idiv-method,
+        rdiv-method,
+        exception-message-attribute,
+        invalid-str-codec,
+        sys-max-int,
+        bad-python3-import,
+        deprecated-string-function,
+        deprecated-str-translate-call,
+        deprecated-itertools-function,
+        deprecated-types-field,
+        next-method-defined,
+        dict-items-not-iterating,
+        dict-keys-not-iterating,
+        dict-values-not-iterating,
+        deprecated-operator-function,
+        deprecated-urllib-function,
+        xreadlines-attribute,
+        deprecated-sys-function,
+        exception-escape,
+        comprehension-escape
+
```

Comment:
I might be wrong but if you want to mimic the [runbot check-style configuration](https://runbot.odoo.com/runbot/build/6162757), you should give this list instead:

```
disable=R,
    W0104,
    W0106,
    W0201,
    W0221,
    W0212,
    W0222,
    W0511,
    W0613,
    W0621,
    W0640,
    W0642,
    W0703,
    W0707,
    E0203,
    E0401,
    E1101,
    E1128,
    E1136,
    C0325,
    C0102,
    C0103,
    C0111,
    C0301,
    C0330,
    C0411,
    I1101
```

Reply:
Wouldn't it make sense to disable=all then enable only what we want, as we do in test_pylint? That's certainly more readable.

Reply:
I agree this would be more readable but the pylint.conf file reflects the runbot configuration

Reply:
@d-fence looks like the same formatted differently, possibly due to the R expanding to lots of stuff? W0222 is signature-differs for instance.

Also it looks like the message categories partially overlap with checkers, so we could disable "refactoring" and that should disable [all the corresponding messages](http://pylint.pycqa.org/en/latest/technical_reference/features.html?highlight=refactoring#refactoring-checker).

Reply:
I find it confusing when you want to check that it matches the runbot config. IMHO it would be simpler and shorter to use the same codes.

Reply:
> I find it confusing when you want to check that it matches the runbot config. IMHO it would be simpler and shorter to use the same codes.

Why not update the runbot configuration though?

Reply:
@d-fence also thought about it afterwards but will the runbot not be using this file as its configuration?